src/blockchains/eth_generator/codepresets.test.ts

```diff
@@ -117,4 +118,19 @@ describe('ETH Contract constructor', () => {
         );
         expect(created).toEqual(EXAMPLE_CONTRACT);
     })
+
+    ;[0, undefined, null].forEach(val => {
+        let obj = <IWideDataType>{ value: 0, name: 'some', hash: 'exist', update: val, life: 1 }
+        ;['update', 'life', 'hash'].forEach(prop => {
+            it(`should fire exception on ${prop} equal to ${val}`, () => {
+                obj[prop] = val;
+                try {
+                    getContractBase('name', [obj]);
+                    expect(false).toBe(true)
```

Comment:
`expect(...).toThrow()` would be more clear
https://jestjs.io/docs/en/expect.html#tothrowerror